### PR TITLE
[TT-4035] GOPATH mode with plugin compiler

### PR DESCRIPTION
**Pull request by jeffy-mathew**
this PR tries to use GOPATH with plugin build

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -110,13 +110,29 @@ jobs:
...
+          export GOPATH=/go
+          mkdir -p /go/src || true
+          whereis go
+          go mod tidy
+          go mod vendor
+          echo "Moving vendor"
+          mv -f vendor/* $GOPATH/src
+          rm -rf vendor
+
+          export GO111MODULE=off
```
Comment (alephnull) [inline .github/workflows/release.yml:124]: This has no effect afaict.

**Review by alephnull — APPROVED** [review]
This looks ok except for one minor comment. So this will be merged to `master`, `release-4.0` and `release-4`?

**Comment by jeffy-mathew** [comment]
> This looks ok except for one minor comment. So this will be merged to `master`, `release-4.0` and `release-4`?

yes

**Review by buger — APPROVED** [review]

**Pull request merged**